cli/js/globals.ts

```diff
@@ -241,6 +242,26 @@ export const windowOrWorkerGlobalScopeProperties = {
   WritableStream: nonEnumerable(writableStream.WritableStreamImpl),
 };
 
+let localStorageIsReady = false;
+export const windowGlobalScopeProperties = {
+  localStorage: getterOnly(() => {
+    // @ts-ignore
+    const { origin } = location;
```

Comment:
`location` was removed: https://github.com/denoland/deno/releases/tag/v1.0.0-rc1

Reply:
Yeah, I'd say there's no way to land anything `localStorage` related without reintroducing `globalThis.location` and first deciding what it means.